modules/webstyle/lib/webstyle_templates.py

```diff
@@ -939,7 +945,7 @@ def detailed_record_container_bottom(self, recid, tabs, ln=CFG_SITE_LANG,
     <br/>
     """ % {'similar' : similar,
            'dates' : creationdate and '<div class="recordlastmodifiedbox" style="position:relative;margin-left:1px">&nbsp;%(dates)s</div>' % {
-                'dates': _("Record created %(x_date_creation)s, last modified %(x_date_modification)s") % \
+                'dates': _("Record added %(x_date_creation)s, last modified %(x_date_modification)s") % \
                 {'x_date_creation': creationdate,
```

Comment:
> So after all I am not convinced this whole branch should really go into master. We could keep it as an OPS branch within INSPIRE as we are going to probably migrate to pu rather than upgrade to any master version in the near future.

:+1: 

> Some of the commits (the 005 ones) though might be nice to preserve.

:+1: 

Can you please amend the branch to remove the "earliest_date" parts and keep only 005 parts, so that #1604 would be closed?  If people can populate `bibrec.creation_date` for batch uploads in this way, then the extra earliest date column indeed does not seem necessary...